besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -3630,10 +3630,17 @@ private String generateConfigurationOverview() {
           .setRpcHttpApis(jsonRpcConfiguration.getRpcApis());
     }
 
-    if (engineJsonRpcConfiguration != null && engineJsonRpcConfiguration.isEnabled()) {
-      builder
-          .setEnginePort(engineJsonRpcConfiguration.getPort())
-          .setEngineApis(engineJsonRpcConfiguration.getRpcApis());
+    if (engineJsonRpcConfiguration != null) {
+      if (engineJsonRpcConfiguration.isEnabled()) {
+        builder
+            .setEnginePort(engineJsonRpcConfiguration.getPort())
+            .setEngineApis(engineJsonRpcConfiguration.getRpcApis());
+      }
+      if (engineJsonRpcConfiguration.isAuthenticationEnabled()
+          && engineJsonRpcConfiguration.getAuthenticationPublicKeyFile() != null) {
```

Comment:
If someone mistakenly doesn't configure it, what will we print? I think there may be some other error so probably ok

Reply:
Prints nothing, engineJsonRpcConfiguration.getAuthenticationPublicKeyFile() will be null

Reply:
ah, it generates one and logs a warning:
`2023-03-29 13:31:29.817+10:00 | main | WARN  | Besu | Engine API authentication enabled without key file. Expect ephemeral jwt.hex file in datadir`

Do you think we should also print out the location of this ephemeral one in the configuration overview?